assessment/services/assessment_event_attempt.py

```diff
@@ -21,24 +22,20 @@ def subscribe_to_assessment_flow(request_data, user) -> TaskGenerator:
     return event.get_task_generator()
 
 
+@catch_exception_and_convert_to_invalid_request_decorator(exception_types=EventDoesNotExist)
```

Comment:
Hello, Jo. If I may ask, do all exception have been extracted to decorators, or just 1 of them?

Reply:
#### Hello, Alex. 
This decorator is only used to catch exceptions that will be converted to `InvalidRequestException`. 

As you can see in the line that you've highlighted, the decorator will catch `EventDoesNotExist `exception and re-throw an `InvalidReqeuestException `with the message thrown by the `EventDoesNotExist `exception.

In some cases where we want to catch multiple exceptions and rethrow them as `InvalidRequestException`, we can use a tuple containing the list of exceptions to be caught as an argument to the `exception_types` parameter. 

For example
`exception_types=(EventDoesNotExists, ValueError)`.

Reply:
Okay Jo, thanks for the explanation.